images/ova/flatcar.yaml

```diff
@@ -0,0 +1,28 @@
+---
+download_images: true
+build_name: "flatcar-lts"
+packer_builder_type: "vsphere"
+guestinfo_datasource_slug: "https://raw.githubusercontent.com/vmware/cloud-init-vmware-guestinfo"
+guestinfo_datasource_ref: "v1.4.0"
+guestinfo_datasource_script: "{{guestinfo_datasource_slug}}/{{guestinfo_datasource_ref}}/install.sh"
+packer:
+  cluster: "zone1"
```

Comment:
remove default value of our test environment.
Generate the override file for flatcar.

Reply:
https://github.com/mesosphere/konvoy-image-builder/pull/880/commits/efff6b226f4dd9e2b659b510970902700442d87b